x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx

```diff
@@ -7,19 +7,38 @@
 
 import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import React from 'react';
+import { InferenceAPIConfigResponse } from '@kbn/ml-trained-models-utils';
 import { isEndpointPreconfigured } from '../../../../utils/preconfigured_endpoint_helper';
 import * as i18n from './translations';
+import { isProviderTechPreview } from '../../../../utils/reranker_helper';
 
 export interface EndpointInfoProps {
   inferenceId: string;
+  provider: InferenceAPIConfigResponse;
 }
 
-export const EndpointInfo: React.FC<EndpointInfoProps> = ({ inferenceId }) => (
+export const EndpointInfo: React.FC<EndpointInfoProps> = ({ inferenceId, provider }) => (
   <EuiFlexGroup justifyContent="spaceBetween">
     <EuiFlexItem grow={false}>
-      <span>
-        <strong>{inferenceId}</strong>
-      </span>
+      <EuiFlexGroup gutterSize="s" alignItems="center">
+        <EuiFlexItem grow={false}>
+          <span>
+            <strong>{inferenceId}</strong>
+          </span>
+        </EuiFlexItem>
+        {isProviderTechPreview(provider) ? (
+          <EuiFlexItem grow={false}>
+            <span>
+              <EuiBetaBadge
+                label={i18n.TECH_PREVIEW_LABEL}
+                size="s"
+                color="hollow"
```

Comment:
Can we change this to "subdued" to help differentiate it more from the preconfigured badges?  Otherwise it's odd that it's not in the same place.